docs/registration.adoc

```diff
@@ -0,0 +1,125 @@
+:toc: left
+:toclevels: 3
+:sectanchors: true
+:sectids: true
+:source-highlighter: rouge
+:icons: font
+
+= Registration
+
+== Stake
+
+Stake T tokens to the Staking Provider account according to the instructions
+received from your Staking Provider or 
+the link:https://docs.threshold.network/guides/migrating-legacy-stakes[Threshold Network Staking Guide].
+
+== Authorize Applications
+
+Authorize the Random Beacon and TBTC applications.
+
+// TODO: Add instruction
+
+[#register-operator]
+== Register Operator
+
+Once the tokens are staked the one controlling the Staking Provider account has to
+register an operator for the Random Beacon and TBTC applications.
+
+If a Key File for the Staking Provider account is available the registration
+can be performed from the <<register-operator-client>>.
+
+Another way is registration from the <<register-operator-web>>.
+
+NOTE: For the Keep Legacy stakes the Staking Provider account and Operator account are
+the same.
+
+[#register-operator-client]
+=== Keep Client
+
+An operator registering transaction can be submitted with the Keep Client.
+
+The commands that should be used are provided below.
+You need to replace few values in the commands:
+
+* `STAKING_PROVIDER_KEY_FILE_ACCOUNT_PASSWORD` - a password used for the Staking Provider
+Ethereum Key File encryption,
+
+* `STAKING_PROVIDER_KEY_FILE_PATH` - a path to the Staking Provider Ethereum
+Key File,
+
+* `ETHEREUM_API_WS_URL` - an URL to the Ethereum WebSocket API,
```

Comment:
There are no such parameters in the below commands.

Reply:
1020f4842a2ed07df0cebc8cf57e0893f5c6dc45

Reply:
e7e25c84751d72301450dc86e24d36af06ce71ab